mne_bids_pipeline/steps/preprocessing/_04_frequency_filter.py

```diff
@@ -218,6 +242,15 @@ def filter_data(
         picks = np.unique(np.r_[picks_regress, picks_artifact, picks_data])
 
     raw.load_data()
+    zapline(
+        raw=raw,
+        subject=subject,
+        session=session,
+        run=run,
+        task=task,
+        fline=cfg.zapline_fline,
```

Comment:
I'm confused. Config says it's OK for `zapline_fline` to be `None` (in fact, default is `None`) but API of `zapline()` func says it must be `float`. And in fact, it's passed directly to `dss.dss_line()` which almost immediately does `sfreq / fline` so passing `None` definitely won't work. But tests are passing, so... what am I missing here?